content/posts/AsynchronousJS.md

```diff
@@ -20,12 +20,12 @@ var test = readSync(file_loc);
 		var test2 = readSync(file_loc2);
 		console.log(test2);
 ```
-This task is done synchronously first file is fetched and test is logged then  
+This task is done synchronously as first file is fetched and test is logged then  
 next file is fetched and logged. What if the file1 is huge file and lets say  
 take **10sec** to be fetched then this become blocking i/o and decreases UX.  
- So to get rid of this situation we can do same task **asynchronously** 
+ So to get rid of this situation we can perform the same task **asynchronously** 
  
- ```javascript
+ ```ecmascript 6
```

Comment:
Change `ecmascript 6` to `javascript` as Hugo does not support syntax highlighting for `ecmascript 6`. 
@geekychaser

Reply:
#5 converted to javascript @rishabhc32